js/join.js

```diff
@@ -173,21 +175,39 @@ function whyRdsPageValidator() {
   return (
     whyRds.value.trim().split(' ').length >= sizeDef.whyRds &&
     foundFrom.value != '' &&
-    whyRds.value.trim() != ''
+    whyRds.value.trim() != '' &&
+    numberOfHours.value != '' &&
+    parseInt(numberOfHours.value) > 1 &&
+    parseInt(numberOfHours.value) <= 100
   );
 }
 
 function dataValidator(element, size) {
-  let counter = document.getElementById(element.id + 'Counter');
-  let words_left = size - element.value.trim().split(' ').length;
-  counter.innerText = `At least, ${words_left} more word(s) required`;
-  if (words_left <= 0) {
-    counter.innerText = '';
-  }
-  if (element.value.trim().split(' ').length >= size && element.value != '') {
-    element.classList.remove('incorrect-data');
+  if (element.id !== 'numberOfHours') {
```

Comment:
Have a separate function for this input, Don't club it with the text input validators.

Reply:
Ok.